### DISCUSSION TITLE
Auto ignintion temperature

### DISCUSSION THREAD

**Discussion by HunterXHunter1997** [firemodels/fds#13355] [Q&A]
Hello team,
I am testing a pool fire extinguishing time, but when I compared the results (simulation and the actual tests) I found a difference between them so I balanced the AIT to have the right extinguishing time.
But It is not right to do that, Becasue in the future when I need to test something that is not tested yet I will not know the AIT that I have to use. 
And as I know the AIT changed when the temp or pressure or the o2 concentration changed 
So can I make change during the simulation or do you think it is fixed and I have to change something else?

**Comment by mcgratta** [comment]
The problem with AIT and other features to predict under-ventilated combustion and/or extinction is grid resolution. Extinction time for a given value of AIT is dependent on grid resolution because the average temperature of a grid cell is dependent on temperature. We're trying to find an engineering algorithm for extinction that relies less on resolved near-flame temperature, but that is a very difficult thing to achieve without being totally ad hoc.
